### ISSUE TITLE
[luci-interpreter] Support Log operator 

### ISSUE THREAD

**Issue by ragmani** [Samsung/ONE#10715]
## What

Let's support Log operator

## Why
When converting LogtSoftmax operator from onnx to circle, onnx.LogSoftmax might be decomposed into onnx.Softmax and onnx.Log.

- onnx
![image](https://user-images.githubusercontent.com/18050611/233903861-14a68d61-c448-459a-b03c-cd93cf025833.png)

- circle
![image](https://user-images.githubusercontent.com/18050611/233903963-e8d33c43-cddd-44a3-9a3d-5b3cc644988c.png)

**Comment by ragmani** [comment]
- tflite kernel
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/kernels/elementwise.cc
```c++
template <typename T>
inline TfLiteStatus EvalImpl(TfLiteContext* context, TfLiteNode* node,
                             std::function<T(T)> func,
                             std::function<TfLiteStatus(T)> validate_input_func,
                             TfLiteType expected_type) {
  const TfLiteTensor* input;
  TF_LITE_ENSURE_OK(context, GetInputSafe(context, node, 0, &input));
  TfLiteTensor* output;
  TF_LITE_ENSURE_OK(context, GetOutputSafe(context, node, 0, &output));
  TF_LITE_ENSURE_TYPES_EQ(context, input->type, expected_type);
  const int64_t num_elements = NumElements(input);
  const T* in_data = GetTensorData<T>(input);
  T* out_data = GetTensorData<T>(output);
  for (int64_t i = 0; i < num_elements; ++i) {
    if (validate_input_func) {
      TF_LITE_ENSURE_OK(context, validate_input_func(in_data[i]));
    }
    out_data[i] = func(in_data[i]);
  }
  return kTfLiteOk;
}

inline TfLiteStatus EvalNumeric(TfLiteContext* context, TfLiteNode* node,
                                float float_func(float)) {
  return EvalImpl<float>(context, node, float_func, kTfLiteFloat32);
}

TfLiteStatus LogEval(TfLiteContext* context, TfLiteNode* node) {
  return EvalNumeric(context, node, std::log);
}
```

- tflite unittests
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/kernels/elementwise_test.cc
```c++
TEST(ElementWise, Log) {
  ElementWiseOpFloatModel m(BuiltinOperator_LOG, {1, 1, 4, 1});
  m.PopulateTensor<float>(m.input(), {1, 3.1415926, 1, 1});
  ASSERT_EQ(m.Invoke(), kTfLiteOk);
  EXPECT_THAT(m.ExtractVector<float>(m.output()),
              ElementsAreArray(ArrayFloatNear({0, 1.14473, 0, 0})));
  EXPECT_THAT(m.GetTensorShape(m.output()), ElementsAreArray({1, 1, 4, 1}));
}


```

**Comment by seanshpark** [comment]
@ragmani , can you fire this PR too?
```diff
diff --git a/compiler/luci-value-test/test.lst b/compiler/luci-value-test/test.lst
index a131ef665..a44c7739b 100644
--- a/compiler/luci-value-test/test.lst
+++ b/compiler/luci-value-test/test.lst
@@ -73,7 +73,7 @@ addeval(LeakyRelu_000)
 addeval(Less_000)
 addeval(LessEqual_000)
 addeval(LocalResponseNormalization_000)
-#addeval(Log_000)
+addeval(Log_000)
 addeval(LogicalAnd_000)
 addeval(LogicalNot_000)
 addeval(LogicalOr_000)
```

**Comment by ragmani** [comment]
Done.
